modules/matrix-operations/test/test_matrix_operations.cpp

```diff
@@ -0,0 +1,426 @@
+// Copyright 2020 Sokolov Andrey
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include "include/matrix_operations.h"
+
+#define ASSERT_NEAR_MATRIX(matrixA, matrixB, threshold) \
+ASSERT_EQ(matrixA.getRows(), matrixB.getRows()); \
+ASSERT_EQ(matrixA.getCols(), matrixB.getCols()); \
+ASSERT_EQ(matrixA.getData().size(), matrixB.getData().size()); \
+ASSERT_EQ(matrixA.getData()[0U].size(), matrixB.getData()[0U].size()); \
+for (int idx{0}; idx < matrixA.getRows(); ++idx) { \
+    for (int jdx{0}; jdx < matrixA.getCols(); ++jdx) { \
+        ASSERT_NEAR(matrixA.getData()[idx][jdx], \
+                    matrixB.getData()[idx][jdx], \
+                    threshold); \
+    }\
+}
+
+TEST(MatrixOperationsTest, Can_Create_Empty_Matrix) {
+    // Act
+    Matrix matrix(26, 12);
+
+    // Assert
+    EXPECT_EQ(26, matrix.getRows());
+    EXPECT_EQ(12, matrix.getCols());
+    EXPECT_NEAR(0.0, matrix.getData()[2U][3U], 0.001);
```

Comment:
```suggestion
    EXPECT_NEAR(0.0, matrix.getData()[2U][3U], 0.001);
``` Нужно проверить всю матрицу